model/src/main/kotlin/config/ReporterConfiguration.kt

```diff
@@ -31,6 +27,5 @@ data class ReporterConfiguration(
      * Reporter specific configuration options. The key needs to match the name of the reporter class, e.g. "FossId"
      * for the FossId reporter. See the documentation of the reporter for available options.
      */
-    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
-    val options: Map<String, Options>? = null
+    val config: Map<String, PluginConfiguration>? = null
```

Comment:
I guess this class should then also get the `@JsonInclude(JsonInclude.Include.NON_NULL)` annotation?